lib/vrl/web-playground/public/index.html

```diff
@@ -1,77 +1,77 @@
 <!DOCTYPE html>
 <html lang="en-US">
-
-<head>
-  <meta charset="utf-8" />
-  <title>VRL playground</title>
-  <link rel="stylesheet" data-name="vs/editor/editor.main"
-    href="https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.20.0/min/vs/editor/editor.main.min.css">
-
-  <style type="text/css">
-    div#input-section {
-      display: grid;
-      grid-template-columns: 50% 50%;
-      margin-bottom: 5rem;
-    }
-
-    div#container-program {
-      border-radius: 25px;
-      overflow: hidden;
-    }
-
-    div#container-event { 
-      border-radius: 25px;
-      overflow: hidden;
-    }
-
-    div#container-output {
-      border-radius: 25px;
-      overflow: hidden;
-    }
-
-    body {
-      font-family: ui-sans-serif,system-ui,-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Helvetica Neue,Arial,Noto Sans,sans-serif,Apple Color Emoji,Segoe UI Emoji,Segoe UI Symbol,Noto Color Emoji;
-    }
-
-  </style>
+  <head>
+    <meta charset="utf-8" />
+    <!-- the below line lets monaco editor to appear nicely in mobile view -->
+    <meta content="width=device-width, initial-scale=1" name="viewport" />
+    <title>VRL playground</title>
+    <link
+      rel="stylesheet"
+      data-name="vs/editor/editor.main"
+      href="https://cdnjs.cloudflare.com/ajax/libs/monaco-editor/0.20.0/min/vs/editor/editor.main.min.css"
+    />
+
+    <link
+      rel="stylesheet"
+      href="index.css"
+    />
+
+    <style type="text/css">
+    </style>
   </head>
 
-<body>
-  <h1>[draft] VRL playground wasm</h1>
-  <p>Open up the developer console and try running</p>
-  <p>run_vrl({"program": ".", "event": {}})')</p>
-  
-  <button onClick="handleRunCode()">run code</button>
-  <button onClick="handleShareCode()">share code</button>
-  <div id="input-section">
-    <div id="cell">
-      <p class="cell-title">Program</p>
-      <div id="container-program" style="height:400px;border:1px solid black;width:800px"></div>
-    </div>
-    
-    <div id="cell">
-      <p class="cell-title">Event</p>
-      <div id="container-event" style="height:400px;border:1px solid black;width:800px"></div>
-    </div>
-    
-  </div>
-  
-  <div id="output-section">
-    <div id="cell">
-      <p class="cell-title">Output</p>
-      <div id="container-output" style="height:400px;border:1px solid black;width:800px"></div>
+  <body>
+    <div id="App">
+      <div id="summary-section">
+        <h1>VRL Playground</h1>
+        <p>
+          We compiled the <a href="https://vrl.dev">Vector Remap Language</a> crate into a web
```

Comment:
This probably isn't the best place to explain how/why "we" did this. Maybe just say something like "You can run VRL programs in the browser".

Reply:
How about "Vector Remap Language (VRL) is an expression-oriented language designed for transforming observability data (logs and metrics) in a [safe](https://vector.dev/docs/reference/vrl/#safety) and [performant](https://vector.dev/docs/reference/vrl/#performance) manner. This playground allows you to run VRL programs straight in the browser.."

Reply:
I think there's two aspects there - one being "What is VRL" and the other being "What can I use the playground for".

Reply:
Changed it to "Vector Remap Language (VRL) is an expression-oriented language designed for transforming observability data. This playground lets you write a program, run it against an event or events, share it, and see how the events are transformed."